src/sst/core/model/python/pymodel_comp.cc

```diff
@@ -374,7 +374,7 @@ static PyObject*
 compSetStatisticLoadLevel(PyObject* self, PyObject* args)
 {
     int              argOK             = 0;
-    uint8_t          loadLevel         = STATISTICLOADLEVELUNINITIALIZED;
+    int              loadLevel         = STATISTICLOADLEVELUNINITIALIZED;
```

Comment:
It's an issue with the python interface.  There's no flag for integers less than 32-bits in the argument parsing code so it was writing 4 bytes to the address of loadLevel instead of only one.  That one was fun to track down.

Reply:
We should check the use of uint8_t for the level in pymodel.cc too then.

Reply:
I found some new documentation that does have codes for parsing smaller integer values.  The instances in pymodel.cc use those codes, though I'm not particularly confident that they will always match up since they use "short int" and "unsigned short int", which I believe are two bytes.  I'm going to make a quick scrub through to make sure that we don't get caught with this later.